_layouts/synapse-default.html

```diff
@@ -86,56 +89,32 @@
     </header>
     <main class="syn-page-main" id="scrollspy-content">
       <div class="syn-container syn-mt-32 syn-mb-48">
-        {{ content | emojify }}
+        <div v-if="areSearchResultsShown">
```

Comment:
Very small nitpick: could we change this to something like `showSearchResults` so it's not quite as wordy?

Reply:
Yeah...that one didn't feel quite right but I couldn't think of anything better. I'll update.